packages/components/src/utils/index.tsx

```diff
@@ -32,6 +31,12 @@ export function useColorScheme(): ColorSchemeName {
   }
 }
 
+/** Returns light/dark theme based on useColorScheme */
+export function useTheme(): Theme {
```

Comment:
Could be a separate ticket, but wonder if we should make unit tests for `index.tsx` as I noticed it's the only thing with poor coverage when working on the unit tests tickets.

Reply:
Created ticket: https://github.com/department-of-veterans-affairs/va-mobile-app/issues/8960